woodwork/logical_types.py

```diff
@@ -216,7 +218,7 @@ class Categorical(LogicalType):
     """
 
     primary_dtype = "category"
-    backup_dtype = "string"
+    backup_dtype = "string[pyarrow]"
```

Comment:
Does setting the `backup_dtype` to `string[pyarrow]` here and in other places cause an issue with `_get_valid_dtype` in the `LogicalType` parent class if a series is determined to be from pyspark? Since the `backup _type` would get returned? I'm sure I'm missing something in the logic of how it works

Reply:
Pyspark does support `string[pyarrow]` dtypes, so this is fine!